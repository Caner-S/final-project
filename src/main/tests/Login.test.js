import React from 'react';
import renderer from 'react-test-renderer';

import LoginForm from "../components/Login";

it('renders correctly when there are no items', () => {
    const tree = renderer.create(<LoginForm />).toJSON();
    expect(tree).toMatchSnapshot();
});