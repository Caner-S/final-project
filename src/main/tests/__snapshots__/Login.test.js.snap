// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly when there are no items 1`] = `
<div>
  <div
    id="firebaseui_container"
  />
</div>
`;
